content/en/user-guide/ci/circle-ci/index.md

```diff
@@ -3,77 +3,437 @@ title: "CircleCI"
+            test -f ls-state.zip && localstack state import ls-state.zip
+...
+  workflows:
+  localstack-build:
+    jobs:
+      - localstack-save-state
+      - localstack-load-state
+```
+More information about state import and export [here](/user-guide/state-management/export-import-state).
+
+#### Cache
+To preserve state between workflow runs, you can take leverage of CircleCI's caching too.
+This strategy will persist Localstack's state for every workflow re-runs, but not for different workflows.
+
+```yaml
+...
+jobs:
+  localstack-update-state:
+    executor: localstack/default
+    steps:
+      ...
+      # Localstack already running
+      # Let's restore previous workflow run's Localstack state
+      - restore_cache:
+          # Use latest "ls-state" prefixed cache
+          key: ls-state-
+      - run:
+          name: Import state
+          command: test -f ls-state.zip && localstack state import ls-state.zip
+      ...
+      # Infrastructure had been updated
+      # Let's update cached Localstack state
+      - run:
+          name: Export state
+          command: localstack state export ls-state.zip
+      - save_cache:
+          key: ls-state-{{checksum ls-state.zip}}
+          paths: ls-state.zip
+  ...
+  localstack-do-work:
+    executor: localstack/default
+    steps:
+      # Localstack already running
+      - restore_cache:
+          # Use latest "ls-state" prefixed cache
+          key: ls-state-
+      - run:
+          name: Import state
+          command: test -f ls-state.zip && localstack state import ls-state.zip
+      ...
+
+
+# Example workflows
+workflows:
+  localstack-build:
+    jobs:
+      - localstack-update-state
+      - localstack-do-work
+      ...
+```
+More information about state management [here](/user-guide/state-management/export-import-state).
+
+#### Cloud Pods
+Cloud pods providing an easy solution to persist Localstack's state, even between workflows or projects.
 
-            echo "Waiting for LocalStack startup..."  
-            localstack wait -t 30                     
-            echo "Startup complete"
-            
+Find more information about cloud pods [here](/user-guide/state-management/cloud-pods/).
+
+##### Multiple projects
+Update cloud pod in it's own project (ie IaC repo).
+
+_Note: If there is a previously created cloud pod, this step can be skipped._
+
+```yaml
+...
+jobs:
+  localstack-update-cloud-pod:
+    executor: localstack/default
+    steps:
+      ...
+      # Localstack already running
+      - run:
+        name: Load state if exists
+        command: localstack pod load <POD_NAME> || true
+      ...
+      # Deploy infrastructure changes
+      ...
+      - run:
+          name: Export state updated state
+          command: localstack pod save <POD_NAME>
+
+
+workflows:
+  localstack-build:
+    jobs:
+      - localstack-update-cloud-pod
+```
+
+In a separate project use the "golden" cloud pod as below:
```

Comment:
> In a separate project use the "golden" cloud pod as below:

What's a golden cloud pod?

Reply:
The term "golden" comes from the AWS Golden AMI concept ([here](https://aws.amazon.com/blogs/awsmarketplace/announcing-the-golden-ami-pipeline/) or [here](https://docs.aws.amazon.com/managedservices/latest/appguide/ex-immute-gold-ami.html)), which is a preconfigured AMI or in this case cloud pod. The source of it is irrelevant here, only important that it exists and available for usage without any necessary configuration or deployment.

Reply:
Okay, would it make sense to define this term before using it? Maybe something like 'create a "golden" cloud pod' in the section where the pod is created?